### DISCUSSION TITLE
Overclocked CPU, RAM & GPU Frequency and clocks are not visible

### DISCUSSION THREAD

**Discussion by cataluna84** [hardinfo2/hardinfo2#122] [General]
The current CPU frequency is not visible on the main/top window and the report that Hardinfo2 generates. Notice the current frequency in the below screenshot. Moreover, can the voltages be extracted from the .so file too?
![Screenshot from 2024-12-23 10-59-40](https://github.com/user-attachments/assets/faee2f73-fde6-46de-9471-3d23f4166455)

The same is the case with RAM frequencies and their voltages as well.
![Screenshot from 2024-12-23 11-02-58](https://github.com/user-attachments/assets/fe057dfc-6f5c-41c0-9253-d5e8e7b1f336)

On the GPU side too, I cannot see the Core and Memory clocks here (It says unknown)
![image](https://github.com/user-attachments/assets/72974cc3-f4d6-4b96-b6b6-9d5d2af66ac2)

**Comment by yankee495** [comment]
Have you done Synchronize to update the hardware ID's etc?
Have you ran lm-sensors to set to your sensors?
What distro are you using?  I'm running Tumbleweed.

Mine does show Ryzen 9 5950X at 5.086 GHz.
The base speed for this CPU is 3.4 GHz with max boot clock of 4.9 GHz.
It is showing the configured speed of 5.086 GHz.

My DDR4 RAM is shown as Type DDR4-2666 (PC4-21300) because all DDR4 RAM is 2666 and anything over that is "overclocked".   Then it shows the XMP Profile which is the advertised speed of the RAM and mine is DDR4 3603 MHz.  When I switched from 3200 to 3600 RAM I gained a little speed because the Ryzen Infinity Fabric clock speed raised to 1800 MHz.  My CPU used to show 5.084 MHz.

But you are correct.  I do not see anywhere Hardinfo2 clearly states the "Configured Memory Speed", although 3603 is my exact RAM speed.  Plasma's Info Center says "Configured Memory Speed: 3600 MT/s", which is the "configured speed" but the actual speed is 3603.

When I click on one of my memory sticks at the top the first line at the bottom says "Source 9-0052 (ee1004)".  The Linux driver ee1004 is the driver responsible for accessing and decoding the Serial Presence Detect (SPD) data stored on the memory modules (DIMMs). This SPD data contains information about the memory's speed, timings, and other characteristics. 

Do the Synchronize if you haven't done that and install and run lm-sensors if you haven't done that.
lm-sensors is the setup program for your sensors to detect temperature and voltage etc.
